spec/requests/pages/questions_controller_spec.rb

```diff
@@ -119,7 +119,7 @@
           mock.get "/api/v1/forms/2/pages", req_headers, form_pages_response, 200
           mock.get "/api/v1/forms/2/pages/1", req_headers, page_response, 200
         end
-
+        draft_question
```

Comment:
Is this line here just for debugging? Or does it do something?

Reply:
Ah, this isn't for debugging. The way lets work in spec is that they are only initialised and set when they are first called. If you look at the code now where does it call `draft_question` before this point and we need a draft_question setup before making the request. If you were to remove this line the tests should break.

I could have used `let!(:draft_question)` rather than `let(:draft_question) which would initialised it when the code was first read but its considered back practice to use let! because you lose the benefit of normal lets only being initialised if they need to and its waste to create the draft question if some specs dont need it.

Reply:
Okay, can we put in a comment explaining what's going on?

Reply:
done and squashed.